frontend/dockerui/config.go

```diff
@@ -54,20 +54,21 @@ const (
 )
 
 type Config struct {
-	BuildArgs        map[string]string
-	CacheIDNamespace string
-	CgroupParent     string
-	Epoch            *time.Time
-	ExtraHosts       []llb.HostIP
-	Hostname         string
-	ImageResolveMode llb.ResolveMode
-	Labels           map[string]string
-	NetworkMode      pb.NetMode
-	ShmSize          int64
-	Target           string
-	Ulimits          []*pb.Ulimit
-	Devices          []*pb.CDIDevice
-	LinterConfig     *linter.Config
+	BuildArgs            map[string]string
+	CacheIDNamespace     string
+	CgroupParent         string
+	Epoch                *time.Time
+	ExtraHosts           []llb.HostIP
+	Hostname             string
+	ImageResolveMode     llb.ResolveMode
+	Labels               map[string]string
+	NetworkMode          pb.NetMode
+	ShmSize              int64
+	Target               string
+	Ulimits              []*pb.Ulimit
+	Devices              []*pb.CDIDevice
+	LinterConfig         *linter.Config
+	ExtraEntrypointFiles []EntrypointFile
```

Comment:
🤔 How is this `ExtraEntrypointFiles` set? `dockerui` is supposed to match the frontend attributes `docker buildx` uses when making requests and this structure is detected based on that. But afaik there is no way you can set multiple `-f` in buildx.